composer.json

```diff
@@ -15,20 +15,20 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
```

Comment:
why drop 7.1 support?

Reply:
:+1: i currently realized that my local ubuntu works on 7.1 only :-)